### ISSUE TITLE
`now` constant appears to be a Date, not a string

### ISSUE THREAD

**Issue by djmitche** [taskcluster/json-e#181]
```yaml
template: now
context: {}
result: 2017-01-19T16:27:20.974Z
```
yet
```yaml
template: {$fromNow: '0 seconds'}
context: {}
result: '2017-01-19T16:27:20.974Z'
```

This should show up as a string value.

**Comment by lauryndbrown** [comment]
Hi, I'd like to work on this issue. But I'm noticing that now is simply giving me the word "now" rather than a date time stamp.

`template = "now"
context = {}
print(jsone.render(template,context))`

When trying it on the docs site (https://taskcluster.github.io/json-e/), I also simply get "now". Is there something special necessary when using the now builtin?

**Comment by djmitche** [comment]
To evaluate a variable, you need to use eval.  My example is wrong.  It should be

```yaml
template: {$eval: now}
context: {}
result: 2017-01-19T16:27:20.974Z
```
sorry about that!

**Comment by djmitche** [comment]
@lauryndbrown as a quick github tip, if you put "Fixes #181" in the description of the PR, then Github will automatically close that issue when the PR is merged.
